src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -60,6 +60,7 @@ const AuditLogsTable = ({ showModel = false }: Props) => {
   const auditLogs = useSelector(getAuditEvents);
   const auditLogsLoaded = useSelector(getAuditEventsLoaded);
   const auditLogsLoading = useSelector(getAuditEventsLoading);
+  const auditLogsLimit = Number(useSelector(getAuditEventsLimit));
```

Comment:
I don't think this needs `Number(...)` does it? I'm pretty sure it should return a number according to the type.